assets/components/ctaLink/ctaLink.jsx

```diff
@@ -12,14 +12,27 @@ type PropTypes = {
   text: string,
   url?: string,
   onClick?: () => void,
+  tabIndex?: number,
 };
 
+// ----- Functions ----- //
+
+const onKeyPressHandler = (handler?: () => void): (event: Object) => void =>
```

Comment:
Could the method name be more informative, e.g.  `clickSubstitueKeyPressHandler` or something?